### ISSUE TITLE
Fonts are sized in px which affects user scaling of text

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#725]
## Description
[WCAG's Resize Text criterion](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html) requires text to be resized up to 200% without loss of content or functionality. **Note that this is just the text, not the entire page.**. Because fonts have been set in `px`, which is an absolute unit, some browsers cannot resize the text at all.

To fix this issue, all `font` and `font-size` rules need to be changed to use relative units. The `em` and `rem` units are good, and [Richard Rutter's idea](http://clagnut.com/blog/2384/) of using `rem`s for global and `em`s for local sizing works well.

### Steps to reproduce
Instructions for Chrome, as I assume that's what most people use:
1. Navigate to Chrome's font settings: chrome://settings/?search=font
2. Select the "Customize Fonts" option
3. Move the font-size slider from `16` to `32`
4. Navigate to a page on the CIVIC site. You'll see that the text is the same size. Then navigate to [BBC News](https://www.bbc.com/news) to see the difference.

### Expected result
What should happen is that the text on the page should scale with the user's scaling preference.

### Actual result
The actual result is that none of the text (that I could find) scaled.

### Environment
All pages I looked at on Civic Platform site.

**Comment by jaronheard** [comment]
@fstorr -- we just merged a PR that fixed this in most(?) places -- if you're able/willing to create a follow-up issue, with further issues, that would be great. Screenshots + referencing this issue is plenty of information. Thanks!

**Comment by fstorr** [comment]
@jaronheard Thanks for working on this. I’m finishing off some documentation on the More/Less links and will circle back on this.
